src/pretalx/orga/templates/orga/cfp/question_view.html

```diff
@@ -45,13 +46,21 @@
                         {{ question.get_target_display }}
                     </td>
                     <td>
-                        <i class="fa fa-{{ question.required|yesno:'check-circle text-success,times-circle text-danger' }}">
-
-                        </i>
+                        {% if question.question_required == "require" %}
+                            {% translate "always" %}
+                        {% elif question.question_required == "require after"%}
+                            {% translate "after" %}
+                        {% elif question.question_required == "freeze after"%}
+                            {% translate "before" %}
+                        {% else %}
+                            {% translate "never" %}
+                        {% endif %}
+                        {% if question.deadline %}
+                            {{ question.deadline|timezone:question.event.timezone }}
```

Comment:
No need to add timezones here, all times are already localised.

Reply:
Fixed🧹🔥